app/client/templates/views/account.js

```diff
@@ -4,11 +4,82 @@ Template Controllers
 @module Templates
 */
 
+/**
+Watches custom events
+
+@param {Object} newDocument  the account object with .jsonInterface
+*/
+var addLogWatching = function(newDocument){
+    var contractInstance = web3.eth.contract(newDocument.jsonInterface).at(newDocument.address);
```

Comment:
Thats the place where it loads the contract with the correct address and starts watching on them... in line 27